src/diffpy/labpdfproc/tools.py

```diff
@@ -42,7 +42,14 @@ def expand_list_file(args):
     the arguments with the modified input list
 
     """
+    wildcard_inputs = [input_name for input_name in args.input if "*" in input_name]
     file_list_inputs = [input_name for input_name in args.input if "file_list" in input_name]
+
```

Comment:
close up blank line

Reply:
I added the test case of having a wildcard pattern in a file-list file. Also, I want to make sure that in the wildcard pattern, we also don't want to read anything related to a file-list file, to avoid confusion? In the test case I just added it is to glob all txt files, that include a file-list file, which I skipped on purpose.